### DISCUSSION TITLE
Funding & Spending

### DISCUSSION THREAD

**Discussion by ysmood** [go-rod/rod#416] [Show and tell]
Set up the funding platform is complicated. Currently, I will use my personal account to manage the donations. If you have any suggestions, please leave a message here. After more trustable developers have joined this project we will move to a more mature model to handle this. To make things as transparent as possible I will post the log of how the funding is used under this thread.

If you think you deserve the sponsorship please leave a comment here, I will put your link [here](https://github.com/go-rod/rod/blob/master/.github/FUNDING.yml).

## Spending Plan

- Set up a demo cluster to serve this [docker image](https://github.com/orgs/go-rod/packages/container/package/rod) so that people can run Rod without installing browsers locally.
- Translate the project into other major languages.
- Reward or hire developers to contribute to this project.

### Sponsor Rod

Click the Sponsor button on this project. Or use the payment method below:

<img width="1830" height="2673" alt="yad-donation" src="https://github.com/user-attachments/assets/737a92df-6f88-4c50-96b3-19a8896e5221" />

**Comment by ysmood** [comment]
Spend 500 CNY on the translation of https://go-rod.github.io into Chinese. The result is here https://go-rod.github.io/i18n/zh-CN

The main translator is https://crowdin.com/profile/shniubobo

**Comment by tivertron** [comment]
Bitcoin +

**Reply by ysmood** [reply]
Thank you for your support!

**Reply by lifesohard** [reply]
是个好项目，就是文档太少啦，对于新手不太友好  感谢

**Comment by ysmood** [comment]
Sponsor logos:

<img width="300" alt="1200_628-1" src="https://github.com/user-attachments/assets/30f786f7-bd42-4ec9-a461-21f0f1d33cf9" />
